homework/style.css

```diff
@@ -1,3 +1,45 @@
+#root {
```

Comment:
I had made more comments on your homework, but somehow they got lost. If I look at your app on a mobile device then the repository values no longer line up vertically with their corresponding labels. Basically, the repository information to be displayed consists of a table with two columns and four rows. So to fix the alignment problem,  I suggest you try an HTML table to replace what you have now.

Your UI:

![image](https://user-images.githubusercontent.com/2788771/65375470-73f2ae00-dc96-11e9-9d57-f01483d251f7.png)


What it could look like using HTML tables:

![image](https://user-images.githubusercontent.com/2788771/65375490-a3a1b600-dc96-11e9-8d43-fcf4b1c95dec.png)

Reply:
Sorry, your comment about being responsive is still there, i did not miss it, however i was working on it.
Since i applied the other one, i immediately wrote you back. Maybe I had to complete all and have sent you both together, sorry for that.

Reply:
Hi, goedemorgen.
I did my best and implemented the feedback about responsiveness.
Hope it works well now.
thanks in advance.